.coveragerc

```diff
@@ -0,0 +1,7 @@
+[paths]
+source = .
+  .tox/*/lib/*/site-packages/
+
+[run]
+source = .
```

Comment:
Should source be ```obal``` so we don't count our tests as coverage?

Reply:
given we call pytest with `--cov={envsitepackagesdir}/obal`, only `obal` is really scanned. but I can try adopting it here too.

Reply:
yeah, works.